### PR TITLE
Galileo DICT-314 updates to terms and terms_enum YAMLs

### PR DESCRIPTION
**Pull request by bullenc**
Updates to terms and enum terms YAML files; changes include additions of new enums and property descriptions/references as well as update to URLs for CDE codes to caDSR II browser.

### CODE REVIEW

**Review by wwysoc2 — APPROVED** [review]
Spot-Checked a few links and they seem to work correctly.  There are way too many lines to comb through, so we'll let the automated checks do the rest of the work.

File: src/gdcdictionary/schemas/_terms.yaml [hunk 1]
```diff
@@ -6967,4 +6966,406 @@ zygosity:
...
+      source:
+      cde_id:
+      cde_version:
+      term_url:
+
+spindle_cell_percent:
+  common:
+    description: >-
+      The percent of uveal melanoma arising from the choroid, ciliary body, or the
+      iris and characterized by the presence of of spindle-shaped melanocytes.
```
Comment (clsackett) [inline src/gdcdictionary/schemas/_terms.yaml:7283]: Typo: "...by the presence of of spindle-shaped melanocytes."
Delete one "of"

**Review by clsackett — COMMENTED** [review]
One typo requiring a fix, otherwise looks good to me

File: src/gdcdictionary/schemas/_terms_enum.yaml [hunk 2]
```diff
@@ -90218,6 +90302,20 @@ stage_i:
         term_url: "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&ns=ncit&code=C112007"
         term_id: C112007
         term_version: 20.05a
+    ensat_pathologic_stage:
+      description: >-
+        Stage I includes: T1, N0, M0. Tumor 5 cm or less in greatest dimension. The
+        tumor has not invaded the surrounding tissues or organs and has not spread
```
Comment (sadhukd) [inline src/gdcdictionary/schemas/_terms_enum.yaml:90308]: The `ensat_pathologic_stage` descriptions are missing the reference (ENSAT 7th Ed, 2009). Not sure if this is absolutely necessary.

**Pull request merged**